x-pack/platform/plugins/shared/triggers_actions_ui/public/application/sections/rule_form/rule_form_route.tsx

```diff
@@ -59,6 +69,37 @@ export const RuleFormRoute = () => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
 
+  const [searchParams] = useSearchParams();
+  const templateId = templateIdParams ?? searchParams.get('fromTemplate') ?? undefined;
+
+  const {
+    data: ruleTemplate,
+    error: ruleTemplateError,
+    isLoading: isLoadingRuleTemplate,
+    isError: isErrorRuleTemplate,
+  } = useRuleTemplate({
+    templateId,
+    enabled: !!templateId,
+  });
+
+  useEffect(() => {
```

Comment:
What is the purpose of this redirection?

Reply:
I removed the redirect as it seems not needed anymore